src/linkPanel/widgets/linking.ts

```diff
@@ -155,16 +172,36 @@ export class Linking extends LinkEditorWidget {
     const output: string[] = [];
 
     this._advancedPanel.node
-      .querySelectorAll('#advanced-link-input select')
+      .querySelectorAll('#advanced-link-INPUT select')
       .forEach(value => {
         input.push((value as HTMLSelectElement).value);
       });
 
     this._advancedPanel.node
-      .querySelectorAll('#advanced-link-output select')
+      .querySelectorAll('#advanced-link-OUTPUT select')
       .forEach(value => {
         output.push((value as HTMLSelectElement).value);
       });
```

Comment:
Could populate the `output` array from models instead of parsing the DOM elements?